File: 3541-find-most-frequent-vowel-and-consonant/3541-find-most-frequent-vowel-and-consonant.java
class Solution {
    public int maxFreqSum(String s) {
        int arr[]=new int[26];
        for(int i=0; i<s.length(); i++){
            arr[s.charAt(i)-'a']++;
        }
        int maxvow=0, maxcon=0;
        for(int i=0; i<26; i++){
            if(i==0 || i==4 || i==8 || i==14 || i==20){
                if(maxvow<arr[i])maxvow=arr[i];
            }else{
                if(maxcon<arr[i])maxcon=arr[i];
            }
        }
        return maxvow+maxcon;
    }
}

File: 3541-find-most-frequent-vowel-and-consonant/3541-find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        maxvow=0
        maxcon=0
        for i in s:
            if(i=='a' or i=='e' or i=='i' or i=='o' or i=='u'):
                if maxvow<s.count(i):
                    maxvow=s.count(i)
            else:
                if maxcon<s.count(i):
                    maxcon=s.count(i)
        return maxcon+maxvow